delay_distributions.Rmd

```diff
@@ -0,0 +1,206 @@
+---
+title: "Delay distributions"
+---
+
+```{r echo = FALSE}
+set.seed(123)
+```
+
+# Objectives
+
+The aim of this session is for you to familiarise yourself with the concept of delay distributions and their use in describing epidemiological processes.
+You will do this by working in `R` with epidemiological time series that are linked via delay distributions.
+
+# Simulating delayed epidemiological data
+
+We will start this session by working with a simulated data set of infections from a disease that has caused an outbreak which subsequently ended.
+For now we will not concern ourselves with the model used to generate the data.
+This represents the typical situation in the real world, where we may have a model of how an infection has spread, but we don't know how well this corresponds to what really happened.
+
+We will later deal with modelling infectious processes.
+For now, we will focus on modelling the *observation* process, in particular the fact that we don't normally observe infections directly but their outcomes as symptomatic cases, hospitalisations or other realisations.
+In particular, we will consider the fact that these observations are *incomplete* (e.g. because not every infection leads to hospitalisations and so focusing on hospitalisations may leave infections unobserved) and happen with a delay (e.g. from infection to symptoms).
+
+We will work with a data set that is included in the `nfidd` R package that you installed initially.
+It can be loaded with the `data` command
+
+```{r}
+library("nfiidd")
+library("ggplot2")
+
+data(infection_times)
+head(infection_times)
+
+## visualise the infection curve
+ggplot(infection_times, aes(x = infection_time)) +
+  geom_histogram() +
+  xlab("Infection time (in days)") +
+  ylab("Number of infections")
+```
+
+We would now like to simulate hospitalisations arising from this outbreak.
+We assume that the incubation period (or time from infection to symptom onset) is gamma-distributed with shape 5 and rate 1.
```

Comment:
This seems way to hard to me? Unless we are doing a lot of work in the lecture to teach what the delay process is

Reply:
What would be the right level of difficulty in your view?

Reply:
One way to simplify this would be to not work with dates but instead numeric days (starting with 0). We could then introduce dates in #54.